bootstrap.sh

```diff
@@ -5,18 +5,44 @@ cd $(dirname "$BASH_SOURCE")
 set -ev
 
 # Update puppet to version >= 3.2.2 before using puppet provisioning.
-wget https://apt.puppetlabs.com/puppetlabs-release-pc1-trusty.deb
-dpkg -i puppetlabs-release-pc1-trusty.deb
+package_name=puppetlabs-release-pc1-trusty.deb
```

Comment:
As per our discussion, https://google.github.io/styleguide/shell.xml#Naming_Conventions are the coding conventions I saw for bash. Constants are supposed to be ALL_CAPS but not variable names.